### PR TITLE
README: Update , merge in Doc/documentation.html

### PR DESCRIPTION
**Pull request by davelab6**

### CODE REVIEW

**Comment by anthrotype** [comment]
thank you Dave! This looks good to me.

File: README.md [hunk 1]
```diff
@@ -2,42 +2,160 @@
...
+```sh
+git clone https://github.com/behdad/fonttools.git ;
+cd fonttools ;
+python setup.py install ;
+```
 
-#### History
+Full build and installation instructions are in [Doc/install.txt](https://github.com/behdad/fonttools/blob/master/Doc/install.txt) 
 
-The fontTools project was started by Just van Rossum in 1999, and was maintained as an open source project at <http://sourceforge.net/projects/fonttools/>. In 2008, Paul Wise (pabs3) began helping Just with stability maintenance. In 2013 Behdad Esfahbod began a friendly fork, thoroughly reviewing the codebase and making changes at <https://github.com/behdad/fonttools> to add new features and support for new font formats.
```
Comment (anthrotype) [inline README.md]: @davelab6 I actually liked this History paragraph.. Any reasons why you decided to remove it?

Comment (miguelsousa) [inline README.md]: I liked it too

**Comment by benkiel** [comment]
+1

File: README.md (hunk 1 above)
Comment (davelab6) [inline README.md]: Oops this got lost in the shuffle! I didn't mean to drop it, and I didn't edit it in my drafting either (other than to put one sentence per line)

File: README.md [hunk 2]
```diff
@@ -2,42 +2,160 @@
 [![Health](https://landscape.io/github/behdad/fonttools/master/landscape.svg?style=flat)](https://landscape.io/github/behdad/fonttools/master)
 [![Coverage Status](https://img.shields.io/coveralls/behdad/fonttools.svg)](https://coveralls.io/r/behdad/fonttools)
 
-### What it is ?
+### What it this?
```
Comment (Pomax) [inline README.md]: Should probably be "What is this?"

File: README.md [hunk 3]
```diff
@@ -2,42 +2,160 @@
...
+Because the order in which glyphs are stored inside the binary font is important, we maintain an ordered list of glyph names in the font.
+
+### Development and feedback
+
+TTX/FontTools development is ongoing, with an active community of developers including professional developers employed at major software corporations and type foundries as well as hobbyists. 
+
+Feature requests and bug reports are always welcome at <https://github.com/behdad/fonttools/issues/>
+
+The best place for discussions about TTX from an end-user perspective as well as TTX/FontTools development is the <https://groups.google.com/d/forum/fonttools> mailing list.
+You can also email Behdad privately at <behdad@behdad.org>
```
Comment (Pomax) [inline README.md]: I hope behdad has decent spam filtering =P

File: README.md [hunk 4]
```diff
@@ -2,42 +2,160 @@
...
+Georg Seifert,
+Miguel Sousa,
+Adam Twardoch,
+Adrien Tétar,
+Vitaly Volkov,
+Paul Wise.
+
+### Copyrights
+
+Copyright (c) 1999-2004 Just van Rossum, LettError (just@letterror.com)
```
Comment (Pomax) [inline README.md]: W\hat happened with the Copyright after 2004?

File: README.md (hunk 3 above)
Comment (davelab6) [inline README.md]: Its all over the web already; https://www.google.com/search?q=behdad%40behdad.org

File: README.md (hunk 4 above)
Comment (davelab6) [inline README.md]: This git history goes back to 1999 - https://github.com/behdad/fonttools/commits/master?page=57 - so its possible to find all the copyright holders with git log grepping; that's how I produced the above list of people acknowledged. 

Probably we should set AUTHORS and CONTRIBUTORS files and change the copyright notice to `Copyright (c) 1999 - 2015, fontTools Project Authors`

File: README.md (hunk 2 above)
Comment (anthrotype) [inline README.md]: https://github.com/behdad/fonttools/commit/b5feb2571bc1d3408b65d9a756858f6c16a8d76e

**Pull request merged**